packages/cheqd-sdk/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@aries-framework/cheqd-sdk",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.1.0",
```

Comment:
Version should be in sync with the other packages (we keep them in sync to make releases and version compatiblity easier for now)